.env

```diff
@@ -0,0 +1 @@
+BROWSER_PATH: 
```

Comment:
Never commit .env file to source. It should be on the ignore list otherwise you might by accident push your secrets.